Bundle/CoreBundle/Builder/ViewCssBuilder.php

```diff
@@ -23,17 +22,17 @@ class ViewCssBuilder
     /**
      * Construct.
      *
-     * @param TemplateMapper $templating
-     * @param                $victoireTwigResponsive
-     * @param                $kernelRootDir
+     * @param EngineInterface $templating
+     * @param                 $victoireTwigResponsive
+     * @param                 $kernelRootDir
      *
      * @internal param WidgetRenderer $widgetRenderer
      */
-    public function __construct(TemplateMapper $templating, $victoireTwigResponsive, $kernelRootDir)
+    public function __construct(Container $container, $victoireTwigResponsive, $kernelRootDir)
```

Comment:
I inject container here to avoid circular reference.
See here : https://github.com/symfony/symfony/issues/2347